src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,121 @@
+
+//Copyright 2017 Google Inc.
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package codeu.controller;
+
+import codeu.model.data.Conversation;
+import codeu.model.data.User;
+import codeu.model.data.Activity;
+import codeu.model.data.Message;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Arrays;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet class responsible for the activity page. */
+public class ActivityFeedServlet extends HttpServlet {
+	
+	private UserStore userStore;
```

Comment:
What editor are you using?  The Google java style guide asks for spaces rather than tabs:

https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

2 spaces for code block indentation and 4 spaces for line continuations.

It seems like a nitpick, but it's important that the team is consistent so that whitespace doesn't look weird and make the code hard to read.

BTW, if you use Eclipse, you can set it up so that it uses spaces instead of tabs:

https://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs

And fix entire files by selecting a region and then asking Eclipse to fix it.